easybuild/easyblocks/g/gamess_us.py

```diff
@@ -188,22 +195,48 @@ def configure_step(self):
 
         self.log.debug("Contents of install.info:\n%s" % read_file(os.path.join(self.builddir, 'install.info')))
 
+        if self.scratch_dir:
+            # FIXME: resolve environment variables used in scratch_dir values
+
+            # use unique subdirectory in specified scratch dir;
+            # we should use a shell command to create this directory,
+            # since scratch_dir may include names of environment variables
+            salt = ''.join(random.choice(ascii_letters) for i in range(10))
+            self.scratch_dir = os.path.join(self.scratch_dir, self.name + '-' + salt)
+
+            # make sure that specified directory is accessible via 'ssh', since rungms relies on that;
+            # touch a file, and then use "ssh localhost ls" to see if it's accessible;
+            # we must use shell commands for this, since scratch_dir may contain names of environment variables
+            test_file = os.path.join(self.scratch_dir, 'test.txt')
+            run_cmd("mkdir -p %s" % self.scratch_dir)
+            run_cmd("touch %s" % test_file)
+            test_cmd = 'ssh localhost "ls %s"' % test_file
```

Comment:
This is not a good thing to do. Think building in a container...
If rungms tries to do any ssh-based stuff it is broken and should be fixed.
I don't think I had that problem back in the days I built this by hand.